specification/keyvault/data-plane/Security.KeyVault.Secrets/routes.tsp

```diff
@@ -0,0 +1,278 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
```

Comment:
We do not want this in the service description. Please create a client.tsp for client codegen customizations.